code-samples/voice/call-log.js

```diff
@@ -8,31 +7,37 @@ USERNAME     = process.env.RC_USERNAME
 PASSWORD     = process.env.RC_PASSWORD
 EXTENSION    = process.env.RC_EXTENSION
 
-var rcsdk = new RC({
+const rcsdk = new RC({
     server:       SERVER,
     clientId:     CLIENTID,
     clientSecret: CLIENTSECRET
 });
-var platform = rcsdk.platform();
+
+const platform = rcsdk.platform();
+
 platform.login({
     username:  USERNAME,
     password:  PASSWORD,
     extension: EXTENSION
 })
 
-platform.on(platform.events.loginSuccess, () => {
-  read_user_calllog()
+platform.on(platform.events.loginSuccess, read_user_calllog);
+
+platform.on(platform.events.loginError, (e) => {
+    console.error(`User login failed : ${e.message}`)
```

Comment:
Writing to stderr so that codechecker-mkdocs can pick it up from the standard error stream and show it in the report